src/libraries/System.Runtime.WindowsRuntime/src/System/IO/StreamOperationAsyncResult.cs

```diff
@@ -296,10 +297,10 @@ private void ThrowWithIOExceptionDispatchInfo(Exception e)
 
     internal class StreamReadAsyncResult : StreamOperationAsyncResult
     {
-        private readonly IBuffer _userBuffer = null;
+        private readonly IBuffer? _userBuffer = null;
```

Comment:
Nit: seems its better to be non null, the only constructor setting it, maybe just remove `null` assignment?